jung-samples/src/main/java/edu/uci/ics/jung/samples/SpatialLensDemo.java

```diff
@@ -97,25 +93,11 @@ public SpatialLensDemo() {
     final VisualizationModel<String, Number, Point2D> visualizationModel =
         new BaseVisualizationModel<>(graph, graphLayoutAlgorithm, preferredSize);
     vv = new VisualizationViewer<>(visualizationModel, preferredSize);
-
-    PickedState<String> ps = vv.getPickedVertexState();
-    PickedState<Number> pes = vv.getPickedEdgeState();
-    vv.getRenderContext()
-        .setVertexFillPaintTransformer(
-            new PickableVertexPaintTransformer<>(ps, Color.red, Color.yellow));
-    vv.getRenderContext()
-        .setEdgeDrawPaintTransformer(
-            new PickableEdgePaintTransformer<>(pes, Color.black, Color.cyan));
+    vv.getRenderContext().setVertexLabelTransformer(Object::toString);
     vv.setBackground(Color.white);
 
     vv.getRenderContext().setVertexLabelTransformer(Object::toString);
 
-    final Function<? super String, Shape> ovals = vv.getRenderContext().getVertexShapeTransformer();
-    final Function<? super String, Shape> squares = n -> new Rectangle2D.Float(-10, -10, 20, 20);
-
-    // add a listener for ToolTips
-    //    vv.setVertexToolTipTransformer(Object::toString);
-
```

Comment:
I took tooltips out of this particular demo to limit some spatial data structure logging about finding nodes.
The QuadTree should also optimize features like tooltips for nodes, but that is not what I am testing in this demo. A (future) RTree will provide similar optimizations for edges.